examples/guides/ordering.ipynb

```diff
@@ -0,0 +1,188 @@
+{
+ "cells": [
+  {
+   "source": [
+    "# Ordering\n",
+    "\n",
+    "This guide is inteded to showcase how you can order an image into UP42 Storage and download it to inspect the result, all with the convenience of the UP42 Python SDK!"
```

Comment:
[...] This guide is **intended** to showcase how you can order an [...]

Reply:
spelling uh